assets/pages/oneoff-contributions/components/paymentMethods.jsx

```diff
@@ -0,0 +1,63 @@
+// @flow
```

Comment:
This `PaymentMethods` component looks really similar to the paymentMethods from monthly-contributions. We should transform this as a shared component and pass different postCheckout function to them